File: MyHeart.Web/ClientApp/models/professionInfo/bonds/SymptomQuestionsDiseaseDto.ts
import DiseaseDto from "../DiseaseDto";

export default class SymptomQuestionsDiseaseDto {
    symptomQuestionsId: number;
    diseaseId: number;
    bondStrength: number;
    disease: DiseaseDto;

    constructor(symptomQuestionsId: number, bondStrength: number, disease: DiseaseDto) {
        this.symptomQuestionsId = symptomQuestionsId;
        this.diseaseId = disease.id;
        this.bondStrength = bondStrength;
        this.disease = disease;
    }
}

File: MyHeart.DTO/ProfessionInformation/SymptomQuestionBond/SymptomQuestions_DiseaseDTO.cs
using MyHeart.DTO.ProfessionalInformation;
using System;
using System.Collections.Generic;
using System.Text;

namespace MyHeart.DTO.ProfessionInformation {
    public class SymptomQuestions_DiseaseDTO : BaseEntityDTO
    {
        public int DiseaseId { get; set; }
        public int SymptomQuestionsId { get; set; }
        public int BondStrength { get; set; }
        public DiseaseDTO Disease { get; set; }
    }
}
